### PR TITLE
Add degree per second unit

### PR DESCRIPTION
**Pull request by tongbong**

### CODE REVIEW

File: UnitsNet/Scripts/UnitDefinitions/RotationalSpeed.json [hunk 1]
```diff
@@ -20,6 +20,23 @@
...
+      {
+        "SingularName": "DegreePerSecond",
+        "PluralName": "DegreesPerSecond",
+        "FromUnitToBaseFunc": "(Math.PI/180)*x",
+        "FromBaseToUnitFunc": "(180/Math.PI)*x",
+        "Prefixes": [ "Nano", "Micro", "Milli" ],
+        "Localization": [
+          {
+            "Culture": "en-US",
+            "Abbreviations": [ "r/s" ]
```
Comment (anjdreas) [inline UnitsNet/Scripts/UnitDefinitions/RotationalSpeed.json]: en-US abbreviation is wrong, isn't it?

File: UnitsNet.Tests/CustomCode/RotationalSpeedTests.cs [hunk 2]
```diff
@@ -42,6 +42,15 @@ public class RotationalSpeedTests : RotationalSpeedTestsBase
 
         protected override double RevolutionsPerSecondInOneRadianPerSecond => 1/(2*Math.PI);
 
+        protected override double DegreesPerSecondInOneRadianPerSecond => (180 / Math.PI) * 1;
+
+        protected override double MillidegreesPerSecondInOneRadianPerSecond => ((180 / Math.PI) * 1)* 1E3;
+
```
Comment (anjdreas) [inline UnitsNet.Tests/CustomCode/RotationalSpeedTests.cs]: I prefer tests having pre-calculated values here, as it's too easy to fall in the trap of just using the reverse conversion expression from the implementation. I recommend you use an external conversion calculator as a sanity check.

Comment (tongbong) [inline UnitsNet.Tests/CustomCode/RotationalSpeedTests.cs]: Should be fixed in commit  e6ba20a.
You are right, I found myself wondering why I was repeating the formula...
Thanks for the tip.

File: UnitsNet/Scripts/UnitDefinitions/RotationalSpeed.json (hunk 1 above)
Comment (tongbong) [inline UnitsNet/Scripts/UnitDefinitions/RotationalSpeed.json]: Should be fixed  in commit 27a01f7.
Thanks for the feedback.

**Comment by anjdreas** [comment]
Perfect!

**Pull request merged**